src/radix.py

```diff
@@ -9,35 +9,88 @@ def count_sort(x: str) -> str:
     >>> count_sort('')
     ''
     """
-    return ""
+    sorted_word = ''.join(sorted(x))
+    return sorted_word
```

Comment:
This is not a good solution. It runs in $O(n\log n)$ where it could run in $O(n)$ if you did an actual count sort.